Create 2021-05-10-Diver-com-paper.md (#158)
subnero1/subnero1.github.io: _posts/2021-05-10-Diver-com-paper.md

---
layout: post

> title: Diver communication and localization paper published

Reviewer: ```suggestion
title: Diver Communication and Localization System using Underwater Acoustics
```

banner : images/banner-pulse-standard.jpg
date:  2021-05-10
categories: wnc; news

> thumbnail: images/pulse-thumbnail-Diver-Comm-05-2021.jpg

Reviewer: Please correct the filename to have all lower case letters

---
<div class='pulse-img-div'>
    <img src="{{site.baseurl}}/images/pulse-thumbnail-Diver-Comm-05-2021.jpg" class='pulse-img'>
</div>

> We are very proud to announce that the our paper about [Diver communication and localization paper](https://ieeexplore.ieee.org/abstract/document/9389462) has been published by IEEE Xplore.

Reviewer: ```suggestion
We are proud to share that the our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462) has been published in IEEE Xplore.
```

> We are proud to share that our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462) has been published in IEEE Xplore.

Reviewer 2: ```suggestion
We are proud to share that our paper titled [_Diver Communication and Localization System using Underwater Acoustics_](https://ieeexplore.ieee.org/abstract/document/9389462){:target="_blank"} has been published in IEEE Xplore.
```

> The paper is the fruit of several months of work across the Subnero team and the collaboration of [Professor Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},

Reviewer: ```suggestion
The paper is the fruit of several months of work by the Subnero team, in collaboration with [Dr. Mandar Chitre](https://arl.nus.edu.sg/people/mandar-chitre/){:target="_blank"}, of the [Acoustic Research Laboratories (ARL)](https://arl.nus.edu.sg/){:target="_blank"},
```

> As the title indicates, the team has developed on an underwater communication system to improve divers safety using our off the shelf software-defined as well as newly designed devices.

Reviewer: ```suggestion
As the title indicates, the team used our software-defined underwater acoustic communication and localization technology to develop a diver communication and localization system to improve operational safety of underwater divers.
```

> The paper was presented virtually by Dr Prasad Anjangi during [Oceans 2020](https://global20.oceansconference.org/) back in October 2020.

Reviewer: ```suggestion
The paper was presented virtually by Dr. Prasad Anjangi during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```

> The paper was presented virtually by Dr. Prasad Anjangi during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.

Reviewer 2: ```suggestion
The paper was presented virtually by [Dr. Prasad Anjangi](https://subnero.com/people/prasad/) during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.
```

> The paper was presented virtually by Dr. Prasad Anjangi during [Global Oceans 2020](https://global20.oceansconference.org/) back in October 2020.

Reviewer 2: Do we have the link for the presentation? If so, maybe that would be more relevant than the given oceans link.

Author: The link, at the moment, is on our Gdrive. Is there a way to share it without giving access to our Drive?

> For more details, dowlonad the paper in the link above or [contact us](info@subnero.com).

Reviewer 2: ```suggestion
Download the full paper in the link above or [contact us](https://subnero.com/contact/) for more details.
```
